packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -1121,16 +1090,12 @@ function* recoverPurchaseIfNecessary() {
       const audioAccountInfo = yield* call(getAudioAccountInfo, {
         tokenAccount
       })
-      const audioBalance = audioAccountInfo?.amount ?? new u64(0)
+      const audioBalance = audioAccountInfo?.amount ?? BigInt(0)
 
       // If the user's root wallet has $AUDIO, that usually indicates a failed transfer
-      if (audioBalance.gt(new u64(0))) {
+      if (audioBalance > BigInt(0)) {
         // Check we can afford to transfer
-        if (
-          new BN(existingBalance)
-            .sub(new BN(rootAccountMinBalance))
-            .gt(new BN(0))
-        ) {
+        if (existingBalance - rootAccountMinBalance > 0) {
```

Comment:
nit: if these are both BigInts, maybe compare to `0n`?

Reply:
Strangely `getBalance` returns a `number` so these are both numbers.